exercises/complex-numbers/complex_numbers.py

```diff
@@ -0,0 +1,3 @@
+class ComplexNumber(object):
+    def __init__(self):
```

Comment:
As it was discussed in #509 it would be a good idea to include expected parameters to not force the user to go through a test class

What are your thoughts on this?

Reply:
Yeah, I think it is a wise thing to do

Reply:
Although, should I also include method signatures? Or only params for constructor? If we don't want people to go through test suite too much, including method signatures makes sense, some tracks here do this.

Reply:
I guess it makes sense to include method signatures since the user uses the provided test suite anyway.
And if someone wants to create their own unique solution, they are free to change provided class placeholder or even test suite (rewrite it, add or remove tests)

I don't see how providing method signatures can influence the user's overall implementation. The most important thing for me is convenience